src/System.Memory/tests/Memory/MemoryManager.cs

```diff
@@ -176,7 +160,20 @@ public static void SpanFromMemoryManagerAfterDispose()
             }
             Assert.Throws<ObjectDisposedException>(() => manager.GetSpan());
         }
-    }
 
+
+        internal class MemoryForInternalMemoryCreate<T> : MemoryManager<T>
```

Comment:
Reuse [CustomMemoryForTest](https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Memory/CustomMemoryForTest.cs) instead of creating a new type?